com.unity.shadergraph/Documentation~/Fade-Transition-Node.md

```diff
@@ -0,0 +1,34 @@
+# Fade Transition Node
+
+## Description
+
+Fade Transition is a method of adding noise to add variation while a function transitions from on to off. This node takes in a fade value and remaps it using the noise value (usually from a texture). When FadeValue is 0, the output is always 0, and when FadeValue is 1, the output is always exactly 1. In between 0 and 1 the transition will follow the pattern in the noise.
```

Comment:
Note: looks like you have started your PR from the branch you previously used. Be careful with that you must start from master branch for reach new feature. Will merge master in this branch

Reply:
Thanks, good to know. I'm a bit of a newb with git.